doc/classes/Color.xml

```diff
@@ -241,38 +244,41 @@
 			<return type="bool" />
 			<param index="0" name="color" type="String" />
 			<description>
-				Returns [code]true[/code] if [param color] is a valid HTML hexadecimal color string. [param color] is not case sensitive, and may be prefixed with a '#' character.
-				For a string to be valid it must be three-digit or six-digit hexadecimal, and may contain an alpha channel value.
+				Returns [code]true[/code] if [param color] is a valid HTML hexadecimal color string. The string must be a hexadecimal value (case-insensitive) of either 3, 4, 6 or 8 digits, and may be prefixed by a hash sign ([code]#[/code]). This method is identical to [method String.is_valid_html_color].
 				[codeblocks]
 				[gdscript]
-				var result = Color.html_is_valid("#55aaFF") # result is true
-				result = Color.html_is_valid("#55AAFF20") # result is true
-				result = Color.html_is_valid("55AAFF") # result is true
-				result = Color.html_is_valid("#F2C") # result is true
-				result = Color.html_is_valid("#AABBC) # result is false
-				result = Color.html_is_valid("#55aaFF5") # result is false
+				Color.html_is_valid("#55aaFF")   # Returns true
+				Color.html_is_valid("#55AAFF20") # Returns true
+				Color.html_is_valid("55AAFF")    # Returns true
+				Color.html_is_valid("#F2C")      # Returns true
+
+				Color.html_is_valid("#AABBC)     # Returns false
+				Color.html_is_valid("#55aaFF5")  # Returns false
 				[/gdscript]
 				[csharp]
-				var result = Color.HtmlIsValid("#55AAFF"); // result is true
-				result = Color.HtmlIsValid("#55AAFF20"); // result is true
-				result = Color.HtmlIsValid("55AAFF); // result is true
-				result = Color.HtmlIsValid("#F2C"); // result is true
-				result = Color.HtmlIsValid("#AABBC"); // result is false
-				result = Color.HtmlIsValid("#55aaFF5"); // result is false
+				Color.HtmlIsValid("#55AAFF");   // Returns true
```

Comment:
Well, this is awkward, this method is actually not public in C# :sweat_smile:.
There is a method in StringExtensions which can be used like this:

```csharp
"#55AAFF".IsValidHtmlColor();
```

But I'd be in favor of exposing `Color.HtmlIsValid` if the method is exposed in GDScript.

Makes me wonder if the method in String is useful, though. We could just redirect users to the method in Color.

Reply:
I can see why it exists in String, too, for consistency with the other "is_valid_*" methods. It's also a pretty shortcut in String. For now it makes sense to have it in both. Although I'm not against steadily moving some _other_ less common methods out of String in the future.

Reply:
I would prefer to have those methods in the respective type as well, if it makes sense, for example `int.IsValid("42")`, and I think the reason why they are not right now probably has to do with static methods not being a thing until recently.

We can always add the static methods to the Variant types since adding API does not break compatibility, but I think we are too late to remove methods from String.

Reply:
Yeah it's too late but it _is_ worth keeping in mind for next time!